app/views/teams/show.html.erb

```diff
@@ -43,6 +43,37 @@
   </tbody>
 </table>
 
+<% if can? :assign_membership_by_email, @team %>
+  <%= link_to t('teams.show.invite_user_to_team'),
+              '#',
+              data: {'toggle': 'modal', 'target': '#invite_user_by_email_modal'},
+              class: 'btn btn-primary' %>
+
+  <div class="modal fade" id="invite_user_by_email_modal" tabindex="-1" role="dialog" aria-labelledby="<%= t('teams.invite_user_to_team.title') %>" aria-hidden="true">
+    <div class="modal-dialog modal-sm" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="<%= t('helpers.links.close') %>"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title"><%= t('teams.invite_user_to_team.title') %></h4>
+        </div>
+        <form action="<%= assign_membership_by_email_team_path(@team) %>" method="post">
+          <%= hidden_field_tag :authenticity_token, form_authenticity_token %>
```

Comment:
@bakoe I had to add this line since the CSRF verifcation of rails would fail otherwise. Do you think we could somehow use the form builder by rails for building this form? The form builder automatically adds this hidden field...

For further reading and other possible solutions:
http://guides.rubyonrails.org/security.html#cross-site-request-forgery-csrf
https://stackoverflow.com/questions/3364492/actioncontrollerinvalidauthenticitytoken

Reply:
quite a chunk! might it be advisable to use a partial template (potentially not only in this case) to clear our view?
http://guides.rubyonrails.org/layouts_and_rendering.html (_3.4 Using Partials_)

Reply:
Yes, good thing to consider... @bakoe what's your opinion?

Reply:
@birneamstiel @tomkellog I like both of your ideas regarding the usage of rails' form builder (maybe we can use inspiration from [#29's implementation](https://github.com/hpi-swt2/sport-portal/pull/228/files#diff-c100cc1e113f1117d1d232ed4956f698R1)) and a partial template (see [#29's implementation](https://github.com/hpi-swt2/sport-portal/pull/228/files#diff-c100cc1e113f1117d1d232ed4956f698) as well) for the modal and the form contained in it.

I will look into this today!